openstack/ironic/templates/etc/_ironic.conf.tpl

```diff
@@ -33,20 +33,17 @@ public_endpoint = https://{{ include "ironic_api_endpoint_host_public" .}}
 
 [keystone]
 auth_section = service_catalog
-region = {{ .Values.global.region }}
 
 [keystone_authtoken]
 auth_section = service_catalog
-memcached_servers = {{ .Chart.Name }}-memcached.{{ include "svc_fqdn" . }}:{{ .Values.memcached.memcached.port | default 11211 }}
 
 {{- include "ini_sections.audit_middleware_notifications" . }}
 
 [service_catalog]
 auth_section = service_catalog
-valid_interfaces = public {{- /* Public, so that the ironic-python-agent doesn't get a private url */}}
-region_name = {{ .Values.global.region }}
-# auth_section
+# ?? valid_interfaces = public {{- /* Public, so that the ironic-python-agent doesn't get a private url */}}
```

Comment:
As it says in the comment.
If you set it to internal, ironic will pass the internal url to the ironic-python-agent.
And the agent then fails, as it is not running in the kubernetes network.